README.md

```diff
@@ -1,5 +1,7 @@
 # Lyef React Component
 [![Build Status](https://travis-ci.org/lyef/lyef-react-component.svg)](https://travis-ci.org/lyef/lyef-react-component/)
+![size](https://badge-size.herokuapp.com/lyef/lyef-react-component/master/dist/Component.min.js.svg)
+![gzip size](https://badge-size.herokuapp.com/lyef/lyef-react-component/master/dist/Component.min.js.svg?compression=gzip)
 
```

Comment:
@cauealves Actually, I don't see any reason for showing the size of `dist/Component.js` 😄 
Regarding the alt, either way for me it's good